microsoft-edge/devtools-guide-chromium/sensors/index.md

```diff
@@ -0,0 +1,230 @@
+
+   ![Selecting 'Portrait upside down' from the 'Orientation' list](./index-images/selecting-portrait-upside-down.png)
+
+* **Custom orientation** to provide your own exact orientation.
+
+   After selecting **Custom orientation**, the **alpha**, **beta**, and **gamma** fields are enabled.  See [Alpha](https://web.dev/device-orientation/#alpha), [Beta](https://web.dev/device-orientation/#beta), and [Gamma](https://web.dev/device-orientation/#gamma) to understand how these axes work.
+
+   After selecting **Custom orientation**, you can also set a custom orientation by dragging the **Orientation Model**.  Hold **Shift** before dragging, to rotate along the alpha axis:
+
+   ![The Orientation Model](./index-images/orientation-model.png)
+
+
+<!-- ====================================================================== -->
+## Force touch events when clicking
+<!-- https://developer.chrome.com/docs/devtools/sensors#touch -->
+
+To test touch events on your website, you can force touch instead of click even if you're testing on a device without a touch screen.
+
+To trigger touch events with your pointer:
+
+1. [Open the Sensors tool](#open-the-sensors-tool).
+
+1. Under the **Touch** drop-down list, select **Force enabled**:
+
+   ![Forcing touch instead of click](./index-images/force-touch.png)
+
+1. Click the **Reload DevTools** button at the top of DevTools.
+
+
+<!-- ====================================================================== -->
+## Emulate idle detector state
+<!-- https://developer.chrome.com/docs/devtools/sensors#idle -->
+
+The [Idle Detection API](https://developer.chrome.com/articles/idle-detection)<!-- "Detect inactive users with the Idle Detection API" --> lets you detect inactive users and react on idle state changes.  With DevTools, you can emulate idle state changes for both the user state and screen state instead of waiting for the actual idle state to change.
+
+To emulate idle states:
+
+1. [Open the Sensors tool](#open-the-sensors-tool).  For this tutorial, you can try it on the [Ephemeral Canvas](https://idle-detection.glitch.me/) demo page.
+
+1. In the demo page, select the **Ephemeral** checkbox, and then click the **Allow** button to grant the demo page the idle detection permission.
+
+   <!-- ![Granting the idle detection permission on a demo page](./index-images/granting-idle-detection.png) can't get dialog again -->
+
+1. Reload the page.
+
+1. From the **Emulate Idle Detector state** drop-down, select one of the following:
+   * **No idle emulation**
+   * **User active, screen unlocked**
+   * **User active, screen locked**
+   * **User idle, screen unlocked**
+   * **User idle, screen locked**
+
+   ![Selecting an idle and locked state on a demo page](./index-images/idle-detection-demo.png)
+
+In this example, DevTools emulates a **User idle, screen locked** state and, in this case, the demo page starts the 60 second countdown to clear the canvas.
```

Comment:
Maybe rephrase to something like "The demo page displays the emulated user and screen states".